### PR TITLE
Network fixed salinity

### PR DESCRIPTION
**Pull request by mcwimm**
New below-ground concepts combining SimpleNetwork and FixedSalinity

### CODE REVIEW

File: Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/NetworkFixedSalinity/c_NetworkFixedSalinity.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Belowground concept combining *FixedSalinity* and *SimpleNetwork*. See corresponding documentations for input parameters.
```
Comment (jbathmann) [inline Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/NetworkFixedSalinity/c_NetworkFixedSalinity.md:1]: Can I use all input parameters from both concepts? this doe not become clear from the text above

File: TreeModelLib/BelowgroundCompetition/FixedSalinity/FixedSalinity.py [hunk 2]
```diff
@@ -32,7 +32,8 @@ def calculateBelowgroundResources(self):
         self.belowground_resources = psi_sali / psi_zero
 
     ## This function reads salinity from the control file.\n
-    def GetSalinity(self, args):
+    def GetSalinity(self, args):  # , missing_tags=None
+        # if not missing_tags:
```
Comment (jbathmann) [inline TreeModelLib/BelowgroundCompetition/FixedSalinity/FixedSalinity.py:36]: incomplete?

**Review by jbathmann — COMMENTED** [review]
Pretty cool inheritance of both classes. Have you tested it?

File: TreeModelLib/BelowgroundCompetition/FixedSalinity/FixedSalinity.py (hunk 2 above)
Comment (mcwimm) [reply TreeModelLib/BelowgroundCompetition/FixedSalinity/FixedSalinity.py:36]: fixed

File: Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/NetworkFixedSalinity/c_NetworkFixedSalinity.md (hunk 1 above)
Comment (mcwimm) [reply Documentation/ProjectFile/MangaProject/tree_dynamics/belowground_competition/NetworkFixedSalinity/c_NetworkFixedSalinity.md]: fixed

**Comment by mcwimm** [comment]
> Pretty cool inheritance of both classes. Have you tested it?

Thanks! And yes, I've tested it and it worked. However, as mentioned in the doc of the basic concept _NetworkBettina_ the growth_output _salinity_ doesn't work with grafted trees because of its reverse calculation.

**Pull request merged**